O/OpenFHE/common.jl

```diff
@@ -0,0 +1,75 @@
+using BinaryBuilder, Pkg
+
+function prepare_openfhe_build(name::String)
+    # Collection of sources required to complete build
+    sources = [
+        GitSource("https://github.com/openfheorg/openfhe-development.git",
+                  "7b8346f4eac27121543e36c17237b919e03ec058"),
+        DirectorySource("./bundled")
+    ]
+
+    # Bash recipe for building across all platforms
+    script = raw"""
+    cd $WORKSPACE/srcdir/openfhe-development/
+
+    # Set proper install directories for libraries on Windows
+    if [[ "${target}" == *-mingw* ]]; then
+      atomic_patch -p1 "${WORKSPACE}/srcdir/patches/windows-fix-cmake-libdir.patch"
+    fi
+
+    mkdir build && cd build
+
+    cmake .. \
+      -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DWITH_BE2=ON \
+      -DWITH_BE4=ON \
+      -DBUILD_UNITTESTS=OFF \
+      -DBUILD_BENCHMARKS=OFF \
+    """ * (name == "OpenFHE_128" ? "-DNATIVE_SIZE=128" : "") *
```

Comment:
IMHO it would be clearer if you always include the `-DNATIVE_SIZE=` argument and set it to either `64` or `128` (maybe by setting a variable above, such that there is less logic here)